### PR TITLE
feat: [PIE-5734]: ElastigroupSetupStep - UI

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
#### Screenshots

https://user-images.githubusercontent.com/98325173/199618098-17c4ca25-f5c0-40ac-aa49-ba9bc51aa5cd.mov

Runtime - 
image

Coverage:
Screenshot 2022-11-04 at 4 41 31 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

File: src/modules/75-cd/components/PipelineSteps/ElastigroupSetupStep/__test__/ElastigroupSetupStep.test.tsx [hunk 1]
```diff
@@ -0,0 +1,439 @@
...
+        target: { value: '4' }
+      })
+      fireEvent.input(queryByNameAttribute(container, 'spec.instances.spec.desired')!, {
+        target: { value: '3' }
+      })
+      fireEvent.input(queryByNameAttribute(container, 'spec.instances.spec.min')!, {
+        target: { value: '2' }
+      })
+      await ref.current?.submitForm()
+    })
```
Comment (rameshx) [inline src/modules/75-cd/components/PipelineSteps/ElastigroupSetupStep/__test__/ElastigroupSetupStep.test.tsx:203]: only `await ref.current?.submitForm()` needs to be wrapped with `act`

File: src/modules/75-cd/components/PipelineSteps/ElastigroupSetupStep/__test__/ElastigroupSetupStep.test.tsx [hunk 2]
```diff
@@ -0,0 +1,439 @@
...
+      const instancesOption = container.querySelectorAll('input[type="radio"]')
+      expect((instancesOption[0] as HTMLInputElement).value).toBe('CurrentRunning')
+      expect((instancesOption[1] as HTMLInputElement).value).toBe('Fixed')
+      expect(instancesOption[1] as HTMLInputElement).toBeChecked()
+      const currentRunningBtn = instancesOption[0] as HTMLInputElement
+      currentRunningBtn.click()
+      expect(instancesOption[0] as HTMLInputElement).toBeChecked()
+
+      await ref.current?.submitForm()
+    })
```
Comment (rameshx) [inline src/modules/75-cd/components/PipelineSteps/ElastigroupSetupStep/__test__/ElastigroupSetupStep.test.tsx:242]: move fireEvent and assertions out of act

File: src/modules/75-cd/components/PipelineSteps/ElastigroupSetupStep/__test__/ElastigroupSetupStep.test.tsx [hunk 3]
```diff
@@ -0,0 +1,439 @@
...
+      <TestStepWidget
+        initialValues={runtimeValuesFixedOpt}
+        type={StepType.ElastigroupSetup}
+        stepViewType={StepViewType.DeploymentForm}
+        template={runtimeValuesFixedOpt}
+      />
+    )
+    expect(getByText('Submit')).toBeTruthy()
+    fireEvent.click(getByText('Submit'))
+  })
```
Comment (rameshx) [inline src/modules/75-cd/components/PipelineSteps/ElastigroupSetupStep/__test__/ElastigroupSetupStep.test.tsx:399]: An assertion can be added after line 398 to assert something that happens after submit.

File: src/modules/75-cd/strings/strings.en.yaml [hunk 4]
```diff
@@ -616,3 +616,11 @@ artifactVersion: 'Version :'
...
 logoOfName: 'Logo of {{name}}'
 deploymentTemplates: 'Deployment Templates'
+ElastigroupStep:
+  appName: App Name
+  minInstances: Min Instances
+  maxInstances: Max Instances
+  desiredInstances: Desired Instances
+  valueCannotBeLessThan: '{{value}} cannot be less than {{value2}}'
+  valueCannotBeGreaterThan: '{{value}} cannot be greater than {{value2}}'
+  valueCannotBe: '{{value}} cannot be empty, string etc. , should only be a positive number'
```
Comment (rameshx) [inline src/modules/75-cd/strings/strings.en.yaml:626]:
```suggestion
  valueCannotBe: '{{value}} cannot be empty, string etc., should only be a positive number'
```

**Review by rameshx — APPROVED** [review]

**Review by rameshx — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Review by rameshx — APPROVED** [review]

**Review by willd0yle — APPROVED** [review]

**Pull request merged**